### DISCUSSION TITLE
Session 3 state

### DISCUSSION THREAD

**Discussion by galliroberto** [MarJose123/filament-lockscreen#62] [Ideas]
If I have two tabs open in lock screen, I unlock one page, on the other I reload the page and go into lock.

You should check the last unlock date, if I refresh and I'm within the lock time it unlocks me automatically

**Comment by MarJose123** [comment]
The issue can be fixed once I migrate the user menu into these changes.
https://filamentphp.com/docs/3.x/panels/navigation#sending-a-post-http-request-from-a-user-menu-item.

I will work on this during my available time.

**Comment by galliroberto** [comment]
I appreciate.

Maybe check also referer.
If referer is lock screen and spent lock time is less redirect.

If I have time I try to do a pull request

**Reply by MarJose123** [reply]
fixed is on the way.

**Comment by MarJose123** [comment]
The issue should be fixed on latest release v2.3
